src/js/components/Form/__tests__/Form-test-controlled.js

```diff
@@ -222,4 +224,65 @@ describe('Form controlled', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('controlled reset', () => {
+    const onSubmit = jest.fn();
+    const onReset = jest.fn();
+    const Test = () => {
+      const [value, setValue] = React.useState({ test: '' });
+      const onChange = React.useCallback(nextValue => setValue(nextValue), []);
+      return (
+        <Grommet>
+          <Form
+            onReset={onReset}
+            onChange={onChange}
+            value={value}
+            onSubmit={onSubmit}
+          >
+            <FormField
+              a11yTitle="test"
+              name="test"
+              required
+              placeholder="test input"
+            />
+            <Button type="reset" primary label="Reset" />
+          </Form>
+        </Grommet>
+      );
+    };
+    const { getByPlaceholderText, getByText, queryByText } = render(<Test />);
+    fireEvent.change(getByPlaceholderText('test input'), {
+      target: { value: 'Input has changed' },
+    });
+    fireEvent.click(getByText('Reset'));
+    expect(onReset).toBeCalledTimes(1);
+    expect(queryByText('Input has changed')).toBeNull();
+  });
+
+  test('form with select', () => {
+    const Test = () => {
+      const [value, setValue] = React.useState('medium');
+      return (
+        <Grommet>
+          <Form>
```

Comment:
Since Form isn't holding either `value` or onChange, should it be on the controlled file test?

Reply:
Good point this test should be in the uncontrolled test file, I will go back and change it